PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -72,6 +73,7 @@ public abstract class PayPalRequest implements Parcelable {
     private final ArrayList<PayPalLineItem> lineItems;
     private final boolean hasUserLocationConsent;
     protected String userAuthenticationEmail;
+    protected String userPhoneNumber;
```

Comment:
@agedd - what are your thoughts on this field name? should we align across platforms?

Reply:
@tdchow `userPhoneNumber` sounds good to me, will make sure to align on iOS as well!